__tests__/e2e/fixtures/test.ts

```diff
@@ -16,6 +16,12 @@ export const test = base.extend<TestFixtures>({
 		await projectSetup.cleanup()
 	},
 	aiAgent: async ({ projectRoot }, use) => {
+		test.skip(
```

Comment:
Every time we use the aiAgent, it's when we want to run these two line of code. So it's better to centralize in the fixture as it's made for that.